.travis.yml

```diff
@@ -51,13 +55,17 @@ jobs:
     stage: docker
     script: make docker/example-cloud-connector
 
+  - name: Build Activiti Cloud Modeling Liquibase
+    stage: docker
+    script: make docker/activiti-cloud-modeling-liquibase
+
   - name: Build Activiti Cloud Modeling
     stage: docker
     script: make docker/activiti-cloud-modeling
 
   - name: Run Acceptance Tests
     stage: test
-    after_script: make delete || echo "Trying to delete helm chart"
+    after_script: echo make delete || echo "Trying to delete helm chart"
```

Comment:
is this a testing leftover?

Reply:
It is temporary to be able to inspect Helm release in k8s.